frontend/ui/lib/widgets/trip_view_widget.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class TripViewWidget extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```

Comment:
I may have done this somewhere that you took this practice from, but I don't think basic composable units should have top level `Center` wrappers. This should just indicate a layout, then the user can wrap it with a `Center` if they want the widget centered in the parent.